### ISSUE TITLE
Integration Test takes forever

### ISSUE THREAD

**Issue by dpc** [fedimint/fedimint#1451]
![image](https://user-images.githubusercontent.com/9209/213840495-29756f87-b81b-48d9-bd34-b0fbbcfd4084.png)

Everything else in our CI pipeline seems to take a reasonable time, except this one item. Any improvement would be a win.

**Comment by jkitman** [comment]
Considering the new wallet tests for electrum/esplora requiring bitcoin locks, perhaps we should move wallet tests to a separate CI task that occurs in parallel?

**Comment by dpc** [comment]
After making these run in parallel it's about 2:30 , so much better. Acceptable for now I guess.

The problem with moving to another task is that tests require workspace to be built, and github doesn't have a good way to first run workspace build once, and *then* diverge somehow. So we would have to build workspace twice, in each CI task. Which is possible, but doesn't seem worth it until just yet.

I might come up with some GNU parallel-based thing to run more thing in parallel eventually.

**Comment by dpc** [comment]
It's reasonable now, everything parallelized.
